docs/source/index.mdx

```diff
@@ -6,7 +6,13 @@
 
 Load a dataset in a single line of code, and use our powerful data processing methods to quickly get your dataset ready for training in a deep learning model. Backed by the Apache Arrow format, process large datasets with zero-copy reads without any memory constraints for optimal speed and efficiency. We also feature a deep integration with the [Hugging Face Hub](https://huggingface.co/datasets), allowing you to easily load and share a dataset with the wider machine learning community.
 
-Find your dataset today on the [Hugging Face Hub](https://huggingface.co/datasets), and take an in-depth look inside of it with the live viewer.
+Find your dataset today on the [Hugging Face Hub](https://huggingface.co/datasets), and take an in-depth look inside of it with the [live viewer](https://huggingface.co/datasets/glue/viewer/cola/train).
+
+<Tip>
+
+Are you looking for the [Datasets REST API](https://huggingface.co/docs/datasets-server)? Integrate into your apps over 10,000 datasets via simple **HTTP requests**, with pre-processed responses and scalability built-in.
+
+</Tip>
```

Comment:
Adding this tip will bring the four blocks "tutorials" "how-to guides" etc. at the very bottom of the page and users will have to scroll. As it is the first page, it must also be minimal and straight to the point. Therefore I'm not a big fan of changing this page.

`datasets` users will probably mostly be interested in having the preview working. I think we can have a dedicated page in the "Dataset Repository" section about "Dataset Preview" that explains how it works, and possibly redirect to the REST API